src/filtered-decks.md

```diff
@@ -184,12 +185,18 @@ Cards return to their home deck when (re)learning is complete. Thus if
 you have 3 learning steps, a new card will return to its home deck upon
 three presses of "Good" or a single press of "Easy".
 
-In the old scheduler, the **custom steps** option allows you to override the
+In the v1 scheduler, the **custom steps** option allows you to override the
 home deck’s steps and provide your own steps instead. The provided steps apply to both
 cards being learnt, lapsed reviews, and reviews ahead of time. Please note, however, that
-in the [new scheduler](https://faqs.ankiweb.net/the-anki-2.1-scheduler.html)
+from the [v2 scheduler](https://faqs.ankiweb.net/the-anki-2.1-scheduler.html)
 filtered decks no longer support custom steps.
 
+### [Scheduler v3](https://faqs.ankiweb.net/the-2021-scheduler.html)
```

Comment:
These headings make the table of contents a bit messy - I would probably opt to make them `**bold**` instead.

<img width="866" alt="Screen Shot 2021-08-03 at 11 28 20 am" src="https://user-images.githubusercontent.com/180542/127943665-574559d0-7682-4c08-900d-38518ea44543.png">

Reply:
Done